Chapters/1-3-Chapter3.md

```diff
@@ -0,0 +1,259 @@
+
+"What we've found at the observatory was too important to nap on."
+
+She sighed, "Well, I'm afraid you're too late. Melna is currently in assembly."
+
+"When will she be out?"
+
+"It can last anywhere from 2 tova'na to the whole day."
+
+"This message is of utmost importance. It will likely affect today's deliberations. Is there any way you can tell her?"
+
+"It's unlawful to interrupt the assembly."
+
+"Surely you can make an exception."
+
+"And I would surely be imprisoned for it."
+
+He sighed in frustration burying his eyes in his hand.
+
+"Pomri," he replied with pessimism. "You can't be serious. Even if there is a law against interrupting the assembly, a magistrate would look the other way at sentencing."
+
+"You ever been to Shaki?"
+
+He hesitated, "Well, no but –"
+
+"– Something to learn before you come. Magistrates get off on being right even if they're wrong. Just wait until the assembly is done."
+
+"But... we can't wait the entire day."
+
+"Look, it sounds like this is really important. Why don't you come to Shaki and we can talk it over?"
+
+"You're asking us to come to the senate? So one of us can do it instead of you?"
+
+"I'm sorry I can't do more, kid. Laws here are really backward but at least I can grumble about 'em. Just think about what I said."
+
+He sighed, "Yea. We- I'll consider it," he said.
+
+"Thanks," she said, and the line went silent.
+
+I built some resolve and double checked that the microphone was off.
+
+"I'll go," I said.
+
+"What are you talking about?"
+
+"I'll interrupt the Senate"
+
+"That's crazy. You heard Pomri. You'll be sent to Ulukoma."
+
+Thimlu butted in. "This is of utmost importance," he said with a smirk on his face. "A magistrate would surely look the other way at sentencing."
+
+Jathi shouted back at him, "Whose side are you on here?!"
+
+I was astonished. Jathi was always so calm and now he was the most animated I had ever seen him. He sighed and sat back, turning it over in his head. I rose from the desk and paced the floor across from him, pausing to admire the old wood floor that countless messengers before him had walked upon.
+
+I tried to convince him again, "I can't think of any better way to do it."
+
+"You are sacrificing too much. The bureaucracy behind the court is impossible."
+
+"It's a sacrifice I'm willing for Liffa to take," said Thimlu. He took the words right out of my mouth.
+
+I explained my thoughts. "One of us has to do it. If this is as important as you say, then there will be a lot of chatter for the next few days. One of you has to keep the communication lines open amidst all that chaos, and one of you has to keep watching the sky for more data. Besides, I have some strings I can pull at the senate. I stand a much better chance."
+
+Jathi seemed defeated but he looked up into my eyes and we locked gazes for a while.
+
+"If you pull this off, I'll petition the magistrate for a lighter sentence."
+
+"Don't worry about me," I told him. "I've got a coat."
```

Comment:
Is it suppose to be an inside joke?

Reply:
Ohh yea this is too early for a joke like this.